src/System.Net.Http/CFNetworkHandler.cs

```diff
@@ -398,25 +397,14 @@ void HandleHasBytesAvailableEvent (object sender, CFStream.StreamEventArgs e)
 
 		void AddCookie (string value, Uri uri, string header)
 		{
-#if NET
-			// .NET: CookieCollection.CookieCutter is internal to mscorlib:
+			// CookieCollection.CookieCutter is internal to mscorlib:
```

Comment:
Do we still need this commment if we no longer use CookieCutter?

Reply:
Good point, removed.